1.9.0/README.md

```diff
@@ -0,0 +1,97 @@
+DirectorOnPrem
+=====================
+
+
+Introduction
+------------
+
+This chart bootstraps DirectorOnPrem on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+- Kubernetes 1.12.0+ with RBAC enabled.
```

Comment:
Will it work for helm both 2 and 3?
If not then we need to add that.

Reply:
I believe this works for both helm 2 and 3. @atulabhi please confirm.

Reply:
yes it will work for both